Packs/FeedGCPWhitelist/Integrations/FeedGCPWhitelist/FeedGCPWhitelist.yml

```
@@ -0,0 +1,100 @@
+category: Authentication
+commonfields:
+  id: GCP Whitelist
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+  defaultvalue: Good
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: A - Completely reliable
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- display: ""
+  name: feedExpirationPolicy
+  defaultvalue: indicatorType
```

```suggestion
  defaultvalue: suddenDeath
```
Because these indicators are going to be assigned a value of `Good`, it is important that these indicators will expire as soon as they are not included in the feed.